sources/core/actions.html

```diff
@@ -1,29 +1,28 @@
-﻿<div class='row er-actions-row'>
-    <div class="actions">
-        <!-- ko foreach: actions -->
-            <!-- ko ifnot: ko.unwrap($data.actions) -->
-                    <!-- ko if: !$data.groupName && !$parent.groupName || $data.groupName == $parent.groupName  -->
-                    <!-- ko component: { name: 'abris-action-item', params: $data } -->
-                    <!-- /ko -->
-                <!-- /ko -->
-            <!-- /ko -->
-            <!-- ko if: ko.unwrap($data.actions) && ko.unwrap($data.actions).length > 0 -->
-            <!-- ko if: !$data.groupName && !$parent.groupName || $data.groupName == $parent.groupName  -->
-
-            <div class="abris-nested-actions abris-nested-actions__title action-context-button">
-                <!--<span class="icon icon-angle-right"></span>
+﻿<div class="actions">
+    <!-- ko foreach: actions -->
+    <!-- ko ifnot: ko.unwrap($data.actions) -->
+    <!-- ko if: !$data.groupName && !$parent.groupName || $data.groupName == $parent.groupName  -->
+    <!-- ko component: { name: 'abris-action-item', params: $data } -->
+    <!-- /ko -->
+    <!-- /ko -->
+    <!-- /ko -->
+    <!-- ko if: ko.unwrap($data.actions) && ko.unwrap($data.actions).length > 0 -->
+    <!-- ko if: !$data.groupName && !$parent.groupName || $data.groupName == $parent.groupName  -->
+    <div class="action__report abris-nested-actions abris-nested-actions__title action-context-button">
```

Comment:
Probably `report` is not a proper word here. Do you mean `abris-nested-actions__container`? We could have a skype call on monday.